src/main/java/org/opentestsystem/ap/imrt/common/model/BaseItem.java

```diff
@@ -591,33 +591,33 @@ public void setTranslationProvided(final String translationProvided) {
     /**
      * @return A collection of ASL files that have been attached to this item
      */
-    public List<Attachment> getAslAttachments() {
+    public Set<Attachment> getAslAttachments() {
         return aslAttachments;
     }
 
-    public void setAslAttachments(final List<Attachment> aslAttachments) {
+    public void setAslAttachments(final Set<Attachment> aslAttachments) {
         this.aslAttachments = aslAttachments;
     }
 
     /**
      * @return A collection of braille files that have been attached to this item
      */
-    public List<Attachment> getBrailleAttachments() {
+    public Set<Attachment> getBrailleAttachments() {
         return brailleAttachments;
     }
 
-    public void setBrailleAttachments(final List<Attachment> brailleAttachments) {
+    public void setBrailleAttachments(final Set<Attachment> brailleAttachments) {
         this.brailleAttachments = brailleAttachments;
     }
 
     /**
      * @return A collection of close-caption files that have been attached to this item
      */
-    public List<Attachment> getCcAttachments() {
+    public Set<Attachment> getCcAttachments() {
```

Comment:
What if we just make these Collection?

Reply:
We could do that.  After discussing offline; with time constraints being what they are, we've agreed to leave this as a `Set` for now.  There'll be a ticket to come back & change it to a `Collection`.